test/e2e/kubectl/kubectl.go

```diff
@@ -847,6 +847,9 @@ metadata:
 
 			// Pod
 			forEachPod(func(pod v1.Pod) {
+				gte, err := framework.KubectlVersionGTE(utilversion.MustParseSemantic("v1.9.0-alpha.0"))
```

Comment:
This is affected by https://github.com/kubernetes/kubernetes/commit/efbfead4ef9fd91ce130586787a6e3c54ca21e02#diff-fde2ce0a73639d19347a5ea6fe001ba6 which is in `v1.9.0-alpha.2`

Reply:
Done

Reply:
Server version should be checked instead